packages/server/graphql/getRateLimiter.ts

```diff
@@ -1,11 +1,41 @@
-import RateLimiter from './RateLimiter'
+import {InMemoryRateLimiter, StubRateLimiter, RateLimiter} from '../utils/rateLimiters'
+
+const RATE_LIMITERS = Object.freeze({
+  'in-memory': InMemoryRateLimiter,
+  stub: StubRateLimiter
```

Comment:
+1 better to use consistent key format: `'stub'`

Reply:
`prettier` actually won't let me do this.

![Screen Shot 2022-02-22 at 08 42 00](https://user-images.githubusercontent.com/630449/155166977-5c63a91d-b481-4a07-bf72-c733de60a861.png)

The reason `in-memory` must be quoted is because it contains a dash.